tests/simulator_test.cpp

```diff
@@ -15,22 +19,13 @@
     FAIL() << "exception " << MESSAGE << " not thrown with expected type "     \
            << #EXCEPTION_TYPE << "'!";                                         \
   }
-
 using constants::kPi;
-using core::Ray;
-using core::RayHitData;
-using core::Vec3;
+using namespace core;
```

Comment:
Not a big deal but, Google C++ Styleguide discourages these, it is better to be explicit in what you use from the core namespace, than import it all.

Reply:
now when I have static variables assigned to class, I can easily shorten all using into three, which is nice :D